src/suggestions.js

```diff
@@ -59,13 +60,21 @@ export default class SimpleteSuggestions extends HTMLElement {
 			let selector = container && container.getAttribute(attr);
 			this[prop] = selector || DEFAULTS[prop];
 		});
+
+		// set aria roles for each option and ensure that it has a valid HTML id
```

Comment:
```suggestion
		// set ARIA roles for each option and ensure that it has a valid HTML ID
```

Arguably this comment is redundant though, as it merely describes the code below?